packages/client/test/db.e2e.spec.ts

```diff
@@ -680,6 +711,28 @@ describe('DatabaseResolver', () => {
       expect(addr).to.be.an('null')
     })
 
+    it('should block writing text record with different chain ID', async () => {
```

Comment:
What's the difference between this test and the one in line 412?

Reply:
one is for subdomains stored on the database and the second one is for 2LD stored on the blockchain